### ISSUE TITLE
English & Arabic channel name the Arabic part is misplaced

### ISSUE THREAD

**Issue by joezak11** [VueTubeApp/VueTube#301]
### Steps to reproduce

1. Open VueTube app
2. Search for Arabic content where the channel's name starts with English text and ends with Arabic text, e.g., "Museum of The Future متحف المستقبل"

### Expected behavior

The channel's name and view count should be displayed next to each other properly.

### Actual behavior

The Arabic part of the channel's name changes places with the view count this does not occur if the channel's name starts with Arabic text and ends with English text.

### VueTube version

build from source commit: e239726 branch: main

### Android version

Android 12

### Other details

![Screenshot_20220615-023845_Pixel Launcher](https://user-images.githubusercontent.com/69003115/173713537-8d75a5f8-84ff-4dab-920e-76ddbf7dd754.png)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to unstable version **[Latest](https://vuetube.app/install/)**.
- [X] I will fill out all of the requested information in this form.

**Comment by Frontesque** [comment]
Is this still an issue?

**Comment by joezak11** [comment]
yep still the same (build from source commit: [fad4bb2](https://github.com/VueTubeApp/VueTube/commit/fad4bb27ddc0ffd2c0744a61fcc5f1e9b8b23ec5))
